### PR TITLE
feat: add link guidance + update skip-to-content guidance

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

- Add guidance for link component
- Update guidance on component pages for skip-to-content links

[Here](https://www.figma.com/file/nfpEteHyEJ9dhJ2YvuvLeo/Documentation-site%2FComponent-Pages?type=design&node-id=12048-34128&mode=design&t=kIOAcR8E15hhOmAV-0) is the Figma file for the latest guidance for links & buttons.

[Zenhub ticket](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/508) fixes https://github.com/cds-snc/design-gc-conception/issues/508

### CODE REVIEW

File: src/en/components/link/base.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+_Also called: anchor, hyperlink, call to action._
+
+A link is a navigational element that brings a person to a new page, website, file, or section on the current page.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Link component preview" %}
+<gcds-link class="me-400" href="#">Link</gcds-link>
```
Comment (ethanWallace) [inline src/en/components/link/base.md:18]: Some of the feedback we received recently is that our examples tend to use the same text content for each element displayed, making it potentially confusing for users using AT. 

Can we add some context to each link (download link, mail to link, etc.)

File: src/en/components/link/code.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Link
+layout: 'layouts/component-documentation.njk'
+translationKey: 'linkCode'
+tags: ['linkEN', 'code']
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/en/components/link/code.md:6]: Will need to uncomment the date

File: src/en/components/link/design.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Link
+layout: 'layouts/component-documentation.njk'
+translationKey: 'linkDesign'
+tags: ['linkEN', 'design']
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/en/components/link/design.md:6]: Same date comment as above

File: src/en/components/link/use-case.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  parent: componentsEN
+  otherNames: anchor, hyperlink, call to action.
+  description: A link is a navigational element that brings a person to a new page, website, file, or section on the current page.
+  thumbnail: /images/common/components/preview-link.svg
+  alt: An abstract representation using a thick blue rectangle above a thin blue line.
+  state: published
+translationKey: 'link'
+tags: ['linkEN', 'usage']
+permalink: /en/components/link/
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/en/components/link/use-case.md:17]: Same date comment as above

File: src/fr/composants/lien/case-dusage.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  parent: componentsFR
+  otherNames: ancre, lien hypertexte, appel à l'action.
+  description: Un lien est un élément de navigation qui amène une personne à une nouvelle page, à un autre site Web, à un fichier ou à une nouvelle section de la page actuelle.
+  thumbnail: /images/common/components/preview-link.svg
+  alt: Une représentation abstraite employant une épaise rectangle bleue en dessus d'une mince ligne bleue.
+  state: published
+translationKey: 'link'
+tags: ['linkFR', 'usage']
+permalink: /fr/composants/lien/
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/fr/composants/lien/case-dusage.md:17]: Same date comment as above

File: src/fr/composants/lien/code.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+title: Lien
+layout: 'layouts/component-documentation.njk'
+translationKey: 'linkCode'
+tags: ['linkFR', 'code']
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/fr/composants/lien/code.md:6]: Same date comment as above

File: src/fr/composants/lien/design.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+---
+title: Lien
+layout: 'layouts/component-documentation.njk'
+translationKey: 'linkDesign'
+tags: ['linkFR', 'design']
+# date: "git Last Modified"
```
Comment (ethanWallace) [inline src/fr/composants/lien/design.md:6]: Same date comment as above

File: src/images/fr/components/anatomy/gcds-link-anatomy.svg
Comment (ethanWallace) [inline src/images/fr/components/anatomy/gcds-link-anatomy.svg:1]: We will need to have this with French text, I think `télécharger` would be proper but better ask an expert :)

**Review by ethanWallace — CHANGES_REQUESTED** [review]
Just a few small things

File: src/en/components/link/base.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+<gcds-link class="me-400" href="long-filename.pdf" download="file.pdf" type="application/pdf">Link</gcds-link>
+<gcds-link class="me-400" href="mailto:test@test.com?subject=Test%20Email">Link</gcds-link>
+<gcds-link class="me-400" href="tel:1234567890">Link</gcds-link>
+{% endcomponentPreview %}
```
Comment (daine) [inline src/en/components/link/base.md:22]: Are we intentionally skipping out the preview for an external link?

File: src/fr/composants/lien/base.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+Un lien est un élément de navigation qui amène une personne à une nouvelle page, à un autre site Web, à un fichier ou à une nouvelle section de la page actuelle.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Aperçu du composant de lien" %}
+<gcds-link class="me-400" href="#">Lien</gcds-link>
+<gcds-link class="me-400" href="long-filename.pdf" download="file.pdf" type="application/pdf">Lien</gcds-link>
+<gcds-link class="me-400" href="mailto:test@test.com?subject=Test%20Email">Lien</gcds-link>
+<gcds-link class="me-400" href="tel:1234567890">Lien</gcds-link>
```
Comment (daine) [inline src/fr/composants/lien/base.md:21]: If we're making changes to the preview, we should update these as well

**Review by daine — CHANGES_REQUESTED** [review]
Just a few comments, but it looks great so far!

File: src/en/components/link/base.md (hunk 8 above)
Comment (melaniebmn) [reply src/en/components/link/base.md:22]: Yes, because there are some debates around having the external icons or not ;)

**Review by ethanWallace — APPROVED** [review]
Ship it!

**Review by daine — APPROVED** [review]
LGTM!

**Pull request merged**